src/coinbase/tests/transaction_test.ts

```diff
@@ -0,0 +1,173 @@
+import { ethers } from "ethers";
+import { Transaction as TransactionModel } from "../../client/api";
+import { Transaction } from "./../transaction";
+
+describe("Coinbase::Transaction", () => {
```

Comment:
```suggestion
describe("Transaction", () => {
```

It seems from other projects using just class name is most idiomatic.
Lets clean these up in a follow up in other files to use `ClassName` to describe top level block.
`#InstanceMethodName` to describe instance methods and `.staticMethodName` to describe static methods.

Reply:
Update this we will have the most of test case refactoring/changes with [PSDK-245]